Makefile

```diff
@@ -13,14 +13,30 @@ export GOBIN=$(GOROOT)/bin/
 export PATH := $(GOBIN):$(PATH)
 GOFMT := $(GOBIN)/gofmt
 GO := $(GOBIN)/go
+KUSTOMIZE := $(GOBIN)/kustomize
+CONTROLLER_GEN := $(GOBIN)/controller-gen
+DEEPCOPY_GEN := $(GOBIN)/deepcopy-gen
+GOVERALLS := $(GOBIN)/goveralls
 
 export KUBECTL ?= cluster/kubectl.sh
 
-all: generate generate-deploy generate-test
+all: generate
 
 $(GO):
 	hack/install-go.sh $(BIN_DIR) > /dev/null
 
+$(KUSTOMIZE): go.mod
```

Comment:
Can you please separate the changes in Makefile to different commit and explain what are the new entries do?

Reply:
I cannot separate them, since it will render the commit not functional, let me explain at this commit I have now.